tests/js/spec/components/notAvailable.spec.tsx

```diff
@@ -8,15 +8,13 @@ describe('NotAvailable', function () {
     expect(screen.getByText('\u2014')).toBeInTheDocument();
   });
 
-  it('renders with tooltip', function () {
-    jest.useFakeTimers();
+  it('renders with tooltip', async function () {
     render(<NotAvailable tooltip="Tooltip text" />);
     expect(screen.getByText('\u2014')).toBeInTheDocument();
     expect(screen.queryByText('Tooltip text')).not.toBeInTheDocument();
 
     userEvent.hover(screen.getByText('\u2014'));
-    jest.advanceTimersByTime(50);
```

Comment:
There's no delay on the tooltip. we don't need timers.

Reply:
Should just not have timers in general, and use a private promise etc. if necessary then await all promises before test end imo